cli/src/scabbard/action/execute.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::{App, Arg, ArgMatches, SubCommand};
+use splinter::service::scabbard::client::ScabbardClient;
+
+use super::CliError;
+
+pub const SUBCMD: &str = "exec";
+const ABOUT_STR: &str = "execute a Sabre contract";
+
+const CONTRACT_ARG: &str = "contract";
```

Comment:
The use of const for all strings seems unnecessary and it's actually hard to read the code here as a result.

Reply:
I second this